erply_api.py

```diff
@@ -67,21 +64,26 @@ class Erply(object):
         ,'getEmployees'
         ,'getProducts'
         ,'getProductCategories'
-        ,'getProductCostForSpecificAmount'     # untested
+        ,'getProductCostForSpecificAmount'  
         ,'getProductGroups'
-        ,'getProductPrices'                    # untested, broken ??
-        ,'getProductPriorityGroups'            # untested
-        ,'getProductStock'                     # untested
+        ,'getProductPrices'               
+        ,'getProductPriorityGroups'          
+        ,'getProductStock'                    
         ,'getProductUnits'
         ,'getPurchaseDocuments'
         ,'getReports'
         ,'getSalesDocuments'
         ,'getServices'
         ,'getWarehouses'
+        ,'getAllowedWarehouses'
```

Comment:
This list is supposed to be alphabetically ordered.

Reply:
Sorry For that